XPNet.GraphicsTest/GraphicsTestPlugin.cs

```diff
@@ -0,0 +1,144 @@
+﻿using XPNet;
```

Comment:
Looking through this test plugin, it looks like it could probably be integrated into the existing test harness (as I described in the comments on XPNetPluginTestHost.cpp) - just write some code that communicates with this plugin, and verifies its results, just like the existing test code in that file communicates with the LoggerPlugin.

Reply:
I have now written the GraphicsPluginTest such that it covers all the new calls. Furthermore, I am starting that plugin together with the test harness now in the same way as you did with the logging plugin.